source/markdown/steekproefontwerp_zandstreek.Rmd

```diff
@@ -1158,32 +1151,32 @@ steekproef_raw <- tar_read(steekproef_thinned, store = targets_store) %>%
     )
   ))
 
-cirkels_Zandstreek <- steekproef_raw %>%
+cirkels_zandstreek <- steekproef_raw %>%
   st_buffer(300)
 ```
 
 ```{r}
 # visualisatie
-mapview(oude_cirkels_sbp,
+mapview(oude_cirkels_buffer,
```

Comment:
Ik zou hier enkel de punten tonen die effectief > 50% overlappen (zoals dat ook gedaan is in het originele bestand van de zandleemstreek). Dan is het wat overzichtelijker waar de punten liggen die we effectief gaan recycleren.

Reply:
verder voorlopig geen opmerkingen (behalve die dat Hans ook al aanhaalde)

Reply:
Aangepast ;)